packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -50,14 +49,6 @@ const BuySell = () => {
         );
     }
 
-    if (buy_sell_store.show_advertiser_page && !buy_sell_store.should_show_verification) {
-        return (
-            <React.Fragment>
-                <AdvertiserPage />
-            </React.Fragment>
-        );
-    }
-
```

Comment:
Moved `AdvertisersPage` to `app-content` since design/QA wants advertisers page to have no tabs

Reply:
Also here, you can just comment this out instead of removing it completely and add the same TODO

Reply:
Okiess